pkg/acceptance/acceptance.go

```diff
@@ -31,210 +32,218 @@ var (
 )
 
 func RunAcceptance(ctx context.Context, logger kitlog.Logger) {
-	// Repeatedly attempt to connect to PgBouncer on the given port, timing out after a
-	// given limit.
-	pgConnect := func(port string) *pgx.Conn {
-		var conn *pgx.Conn
-
-		defer func(begin time.Time) {
-			logger.Log("event", "pg.connect", "msg", "connected to PostgreSQL via PgBouncer",
-				"elapsed", time.Since(begin).Seconds())
-		}(time.Now())
-
-		Eventually(
-			func() error {
-				logger.Log("event", "pg.connect.poll", "msg", "attempting to connect to PostgreSQL via PgBouncer")
-				cfg, err := pgx.ParseConnectionString(
-					fmt.Sprintf(
-						"user=postgres dbname=postgres host=localhost port=%s "+
-							"connect_timeout=1 sslmode=disable",
-						port,
-					),
-				)
-
-				Expect(err).NotTo(HaveOccurred())
-				conn, err = pgx.Connect(cfg)
-				return err
-			},
-			time.Minute,
-			time.Second,
```

Comment:
We had set the POLLING_INTERVAL (to 1s) as the default polling interval is set to 10ms which is extremely noisy in the test logs while it was waiting for the db connection to come up. I think you have added a wait somewhere so perhaps this isn't annoying anymore.

If I see this as I get further through the PR that this is no longer an issue, I'll delete this comment.